src/Widgets/CameraView.vala

```diff
@@ -20,10 +20,11 @@
  *              Corentin Noël <corentin@elementary.io>
  */
 
-public class Camera.Widgets.CameraView : Gtk.Stack {
+public class Camera.Widgets.CameraView : Gtk.Container {
```

Comment:
You have to use a non-abstract class (e.g. Gtk.Box) for this to work.  On OS6.1, subclassing Gtk.Container compiles but running the flatpak gives the error:
```
jeremy@Kratosaef3cce6:~/Documents/Elementary/camera$ flatpak run io.elementary.camera 
Gtk-Message: 16:44:47.497: Failed to load module "canberra-gtk-module"
Gtk-Message: 16:44:47.498: Failed to load module "canberra-gtk-module"

(io.elementary.camera:2): Gtk-WARNING **: 16:44:47.525: GtkContainerClass::add not implemented for 'CameraWidgetsCameraView'
**
Gtk:ERROR:../gtk/gtkwidget.c:12462:gtk_widget_real_realize: assertion failed: (!_gtk_widget_get_has_window (widget))
Bail out! Gtk:ERROR:../gtk/gtkwidget.c:12462:gtk_widget_real_realize: assertion failed: (!_gtk_widget_get_has_window (widget))
```